content/en/02-installing-pixie/02-setting-up-k8s/03-gke-setup.md

```diff
@@ -39,10 +39,10 @@ gcloud config set compute/zone compute-zone
 
 ### Create the cluster
 
-Run the following command to create a 2 node cluster:
+Run the following command to create a 2 node cluster (each node with 2 vCPUs and 12GB of RAM):
 
 ```
-gcloud container clusters create cluster-name --num-nodes=2
+gcloud container clusters create cluster-name --num-nodes=2 --machine-type=custom-2-12288
```

Comment:
If we don't want a non-custom type, I believe `e2-standard-2` should be sufficiently large and a tiny bit cheaper since its 8GB of RAM instead of 12GB.

Reply:
I haven't tested this yet though.